src/storage.js

```diff
@@ -0,0 +1,28 @@
+const storage = window.localStorage;
+
+export const setItem = (key, value) => {
+  try {
+    storage.setItem(key, JSON.stringify(value));
+  } catch (e) {
+    console.log(e);
+  }
+};
+
+export const getItem = (key, defaultValue) => {
+  try {
+    const storedValue = storage.getItem(key);
+
+    if (!storedValue) {
+      return defaultValue;
+    }
+
+    const parsedValue = JSON.parse(storedValue);
+    return parsedValue;
```

Comment:
```suggestion
    return JSON.parse(storedValue) ?? defaultValue;
```
저도 멘토님에게 배운 내용이지만 null 병합 연산자를 사용해보시면 더 간결하게 작성이 가능합니다!

Reply:
TodoList 때 지적할걸 봤는데 바로 적용하지 못했네요..😅
간결한 코드 작성하도록 노력하겠습니다ㅎㅎ